test/python/circuit/library/test_mcmt.py

```diff
@@ -267,6 +268,19 @@ def test_vchain_open_control(self):
             )
         )
 
+    def test_mcmt_x_gate(self):
+        """Test MCMT gate uses `synth_mcmt_x` for X gate synthesis."""
+        for num_ctrl, num_targ in product(range(1, 6), range(2, 6)):
+            mcmt_x_gate = MCMTGate(XGate(), num_ctrl_qubits=num_ctrl, num_target_qubits=num_targ)
+            qc = QuantumCircuit(num_ctrl + num_targ)
+            qc.compose(mcmt_x_gate, range(num_ctrl + num_targ), inplace=True)
```

Comment:
`append` would be to more appropriate (and faster) method to call here I think, since it's especially for `Instruction`s, whereas `compose` is more generic. You can leave as is, but just for the future 🙂

Reply:
Done